jvcl/run/JvDBGrid.pas

```diff
@@ -4320,6 +4320,7 @@ procedure TJvDBGrid.DoAutoSizeColumns;
   try
     // Get useable width
     ColLineWidth := Ord(dgColLines in Options) * GridLineWidth;
+    ColLineWidth := PPIScale(Self, ColLineWidth);
```

Comment:
Why introduce a new assignment when the call to `PPIScale` could be made inline, just like what's done elsewhere?